cli/EAP7-1113_Disable_CLI_paging_with_an_option.adoc

```diff
@@ -0,0 +1,69 @@
+= EAP7-1113 Disable CLI paging with an option
+:author:            Marek Marusic
+:email:             mmarusic@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          cli,disable,option,output,scrolling
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The output paging is not desired in some cases e.g. when copy/pasting the full output of a command.
+The paging could be disabled by a simple option.
+
+=== Workaround
+It is currently possible to override the paging of a single command by redirecting the output to /dev/stdout.
+
+==== Example of the workaround:
+----
+/subsystem=elytron:read-resource-description(recursive=true) > /dev/stdout
+----
+
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-1113[EAP7-1113]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/WFCORE-4127[WFCORE-4127]
+* https://issues.jboss.org/browse/WFCORE-4096[WFCORE-4096]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+* mailto:jdenise@redhat.com[Jean-François Denise]
+
+=== QE Contacts
+
+* mailto:mnovak@redhat.com[Miroslav Novák]
+
+=== Affected Projects or Components
+
+* CLI
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+* Ability to disable output paging via CLI argument e.g.:
+----
+./jboss-cli.sh --no-paging
```

Comment:
@MMarus  @jfdenise I think users will need to change this behaviour per command basis. Like:
```
[standalone@localhost:9990 /] /subsystem=elytron:read-resource-description(recursive=true) --no-paging
```
or 
```
./jboss-cli.sh -c 
[standalone@localhost:9990 /] disable-paging
[standalone@localhost:9990 /] /subsystem=elytron:read-resource-description(recursive=true)
```
At moment of starting CLI, I don't think that users plans to disable paging. At least I don't :-) wdyt?

Reply:
There is a workaround for this:
```/subsystem=elytron:read-resource-description(recursive=true) > /dev/stdout```
however, it might not be the best solution for this. Let's wait for @jfdenise 's opinion about this :).

Reply:
The RFE is really to "start" the cli with paging ON or OFF, not to enable/disable per command.